sdk/network/Microsoft.Azure.Management.Network/src/Generated/Models/ExpressRouteConnection.cs

```diff
@@ -45,20 +45,17 @@ public ExpressRouteConnection()
         /// connection.</param>
         /// <param name="enableInternetSecurity">Enable internet
         /// security.</param>
-        /// <param name="expressRouteGatewayBypass">Enable FastPath to vWan
-        /// Firewall hub.</param>
         /// <param name="routingConfiguration">The Routing Configuration
         /// indicating the associated and propagated route tables on this
         /// connection.</param>
-        public ExpressRouteConnection(ExpressRouteCircuitPeeringId expressRouteCircuitPeering, string name, string id = default(string), string provisioningState = default(string), string authorizationKey = default(string), int? routingWeight = default(int?), bool? enableInternetSecurity = default(bool?), bool? expressRouteGatewayBypass = default(bool?), RoutingConfiguration routingConfiguration = default(RoutingConfiguration))
+        public ExpressRouteConnection(ExpressRouteCircuitPeeringId expressRouteCircuitPeering, string name, string id = default(string), string provisioningState = default(string), string authorizationKey = default(string), int? routingWeight = default(int?), bool? enableInternetSecurity = default(bool?), RoutingConfiguration routingConfiguration = default(RoutingConfiguration))
             : base(id)
```

Comment:
This is a breaking change. You removed a boolean here hence anyone passing 9 parameters into this method will be broken